src/constants/api.js

```diff
@@ -27,26 +27,26 @@ import { isBundled, RETURN_PAGE_TYPE } from './variables'
   API_BASE_URL: 'https://web-range-myra-prod.pathfinder.gov.bc.ca/api'
 }*/
 
-const DEV_API_BASE_URL = 'https://web-range-myra-dev.pathfinder.gov.bc.ca/api'
-// const DEV_API_BASE_URL = 'http://localhost:8000/api';
+//const DEV_API_BASE_URL = 'https://web-range-myra-dev.pathfinder.gov.bc.ca/api'
+const DEV_API_BASE_URL = 'http://localhost:8080/api'
 const DEV = {
   // eslint-disable-line no-unused-vars
   SSO_BASE_URL: 'https://sso-dev.pathfinder.gov.bc.ca',
   SITEMINDER_BASE_URL: 'https://logontest.gov.bc.ca',
   API_BASE_URL: DEV_API_BASE_URL
 }
 
-/*const TEST = {
+const TEST = {
   // eslint-disable-line no-unused-vars
   SSO_BASE_URL: 'https://sso-test.pathfinder.gov.bc.ca',
   SITEMINDER_BASE_URL: 'https://logontest.gov.bc.ca',
   API_BASE_URL: 'https://web-range-myra-test.pathfinder.gov.bc.ca/api'
-}*/
+}
 
 export const DEV_ENV = {
   // ...PROD,
-  ...DEV
-  // ...TEST,
+  //.DEV
+  ...TEST
 }
```

Comment:
Can you remove the changes in this file? This is just your dev config, and we don't need to commit it to the repo.

Reply:
Hey @calebissharp, thanks for pointing that out. I have changed that in this commit de22be0